cmd/pinniped-proxy/Dockerfile

```diff
@@ -16,7 +16,9 @@ RUN cargo build --release
 FROM bitnami/minideb:buster
 RUN apt-get update && apt-get install -y ca-certificates libssl1.1 && rm -rf /var/lib/apt/lists/*
 COPY --from=builder /pinniped-proxy/target/release/pinniped-proxy /pinniped-proxy
+ENV PATH="/:$PATH"
 
 EXPOSE 3333
 USER 1001
-CMD ["/pinniped-proxy"]
+ENTRYPOINT [ "pinniped-proxy" ]
+CMD [ "--help" ]
```

Comment:
are you sure you want to leave this CMD?

Reply:
Yeah, only because it's what the bitnami container does, so I want to do the same so that we ensure we're setting the command explicitly.